.travis.yml

```diff
@@ -67,6 +67,29 @@ matrix:
         - TESTMODE=fast
         - COVERAGE=
         - NUMPYSPEC=numpy
+  allow_failures:
+    - python: 2.7
+      env:
+        - TESTMODE=fast
+        - REFGUIDE_CHECK=1
```

Comment:
This allows the whole run to fail, correct? Not only linkchecker, but also refguide-check, and the test run with relaxed strides checking.